src/openmrs-esm-login.tsx

```diff
@@ -12,3 +13,5 @@ const lifecycles = singleSpaReact({
 export const bootstrap = lifecycles.bootstrap;
 export const mount = lifecycles.mount;
 export const unmount = lifecycles.unmount;
+
+export { UserHasAccessReact };
```

Comment:
why are you exporting `UserHasAccessReact` here? Isn't is exported from `user-has-access-react.component.tsx`?

Reply:
I could be wrong but my understanding is that only the items exported from this file will be subsequently imported in the compiled version. Let me double check this.